src/get_services_by_team.py

```
@@ -0,0 +1,24 @@
+    """
+    response = requests.get(f"{api_url}/teams/{team_id}/services")
+    response.raise_for_status()
+    return response.json()
+
+def _fetch_all_teams(api_url: str) -> list:
+    """
+    Returns all teams from the service atlas api
+    :param api_url:
+    :return:
+    """
+    response = requests.get(f"{api_url}/teams")
```

Ruff (0.14.6)

Probable use of `requests` call without timeout

(S113)